lib/monmon.rb

```diff
@@ -3,10 +3,39 @@
 require 'net/http'
 require 'json'
 
-url = "https://api.exchangerate.host/latest?base=BYN"
-uri = URI(url)
-response = Net::HTTP.get(uri)
-RATES = JSON.parse(response)
+def rates
+  @rates ||= begin
+    config = Configuration.instance
+    url = "https://api.exchangerate.host/latest?base=#{config.main_currency}"
+    uri = URI(url)
+    response = Net::HTTP.get(uri)
+    JSON.parse(response)
+  end
+end
+
+class Configuration
+  attr_accessor:main_currency
+
+  def initialize
+    @main_currency = :BYN
+  end
+
+  def self.instance
+    @@instance
+  end
+
+  def main_currency
+    @main_currency
+  end
+  def change_main_currency(currency)
+    @main_currency = currency
+  end
+
+  @@instance = Configuration.new
```

Comment:
class variables (starting from `@@`) has disadvantages:
- there are problems with [inheritance](https://stackoverflow.com/questions/40819068/why-is-using-a-class-variable-in-ruby-considered-a-code-smell)
- there are problems using them [in multi-threaded environment](https://stackoverflow.com/questions/9396563/why-should-we-avoid-using-class-variables-in-rails)